RussianFilter/sections/adservers.txt

```diff
@@ -1,6 +1,7 @@
 !
 ! Содержит доменные имена, использующиеся рекламными сетями.
 !
+||ipqolj.com^$third-party
```

Comment:
not enough. Add to `Lucky ads` rule.

Reply:
https://github.com/AdguardTeam/AdguardFilters/commit/c434720f1adaa0a12bb3440af53dabf969ffe8ab